airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java

```diff
@@ -77,9 +77,9 @@ public ConnectionRead createConnection(ConnectionCreate connectionCreate) throws
         .withStatus(toPersistenceStatus(connectionCreate.getStatus()));
 
     if (connectionCreate.getSyncSchema() != null) {
```

Comment:
maybe worth adding a todo here that this behavior is probably not deseriable. i don't think sending a null configured catalog should be valid. not the right time to change this but let's leave ourselves a breadcrumb so we can come back to it.